### PR TITLE
Added automatic testing on pull request

### PR DESCRIPTION
**Pull request by maxlin94**

### CODE REVIEW

**Review by AlbinNils98 — APPROVED** [review]
Nice job!

**Review by AntonisTerzo — APPROVED** [review]
Good job Max

**Review by oskber — APPROVED** [review]

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [21.x, 20.x, 19.x]
```
Comment (richardolsson) [inline .github/workflows/node.js.yml:19]: This is a nice way to test your code on multiple Node versions, which is very good if you're writing code that needs to work on multiple versions, e.g. a library.

For an application like this, there is no real need to test on multiple node versions. It just means the tests take three times as long to run.

**Review by richardolsson — COMMENTED** [review]
Nice work! One minor comment below.

**Pull request merged**